config.lua

```diff
@@ -74,6 +74,12 @@ add_setting("UI", "background_preview", true, {
     end,
 })
 add_setting("UI", "background_preview_has_text", false, { dependencies = { background_preview = true } })
+add_setting("UI", "in-game_buttons", true, {
+    onchange = function()
+        local buttons = require("ui.screens.game.controls")
+        buttons.hidden = not buttons.hidden
+    end
+})
```

Comment:
You don't have to make an onchange function. In this case i'd recommend just getting the config option at the point where new buttons are added in `input.lua` .